uint8 armor_id
uint8 hurt_type